pkg/adapter/config/kubernetes/sensitiveDoguConfigRepository.go

```diff
@@ -0,0 +1,45 @@
+package kubernetes
+
+import (
+	"context"
+	"fmt"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/domain/common"
+	"github.com/cloudogu/k8s-registry-lib/config"
+	"github.com/cloudogu/k8s-registry-lib/repository"
+)
+
+type SensitiveDoguConfigRepository struct {
+	repo repository.DoguConfigRepository
+}
+
+func (e SensitiveDoguConfigRepository) GetAll(ctx context.Context, doguNames []common.SimpleDoguName) (map[common.SimpleDoguName]config.DoguConfig, error) {
+	var configByDogus map[common.SimpleDoguName]config.DoguConfig
+	for _, doguName := range doguNames {
+		loaded, err := e.Get(ctx, doguName)
+		if err != nil {
+			return nil, fmt.Errorf("could not load sensitive config for all given dogus: %w", err)
+		}
+		configByDogus[doguName] = loaded
```

Comment:
🚫 **[checkstyle]** <staticcheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>SA5000: assignment to nil map
<!-- __reviewdog__:ChBlMjEwNDc5MmY4NTA0NzNmEgpjaGVja3N0eWxl -->